x/oracle/types/params_test.go

```diff
@@ -28,10 +29,10 @@ func TestValidateVoteThreshold(t *testing.T) {
 	require.ErrorContains(t, err, "invalid parameter type: string")
 
 	err = validateVoteThreshold(sdk.MustNewDecFromStr("0.31"))
-	require.ErrorContains(t, err, "vote threshold must be bigger than 33%: 0.310000000000000000")
+	require.ErrorContains(t, err, "threshold must be bigger than 0.330000000000000000 and <= 1")
```

Comment:
We shouldn't need this test anymore with the new one at the end of the file covering these cases

Reply:
ok, will remove this one.